templates/hoover-workers.nomad

```diff
@@ -76,6 +77,7 @@ job "hoover-workers" {
       env {
         SNOOP_ES_URL = "http://{% raw %}${attr.unique.network.ip-address}{% endraw %}:9990/_es"
         SNOOP_TIKA_URL = "http://{% raw %}${attr.unique.network.ip-address}{% endraw %}:9990/_tika/"
+        SNOOP_THUMBNAIL_URL = "http://{% raw %}${attr.unique.network.ip-address}{% endraw %}:9990/_thumbnail/"
```

Comment:
Let's make this one optional too, with some `_enabled` config flag like the one for PDF previews below. By default it's false to make it backwards compatible.